docs/source/clientcli.rst

```diff
@@ -41,7 +41,7 @@ Fabric-CA Client's CLI
           --id.name string                 Unique name of the identity
           --id.secret string               The enrollment secret for the identity being registered
           --id.type string                 Type of identity being registered (e.g. 'peer, app, user') (default "client")
-          --loglevel string                Set logging level (info, warning, debug, error, fatal, critical)
+          --loglevel string                Set loggi ng level (info, warning, debug, error, fatal, critical)
```

Comment:
Why are we adding a space here

Reply:
If its what you say I love it, especially later in the summer